### PR TITLE
2022.07.18.notebook template

### PR DESCRIPTION
**Pull request by Samuel-Egbert31415**
# Description

Added post-hoc visualization to the notebook experiment after the model has been changed.  Also added the supplementary files needed for the experiment to run.  The changes to the notebook are reflected in template file and the init.py. Notably, this is only to the Kaggle test data.  Loom Video: https://www.loom.com/share/19285a5968584d72ae5ffb589d84a7e8

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by caitlinkermen — DISMISSED** [review]
Very thorough, after reading through your changes I do not see any issues.

**Review by yestrella14 — DISMISSED** [review]
well detailed

File: structured_experiments/2022.07.19.notebook_template/experiment.MD [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!--DESC Template folder to use for experiments-->
+ Hypothesis
+TODO: Create Template
```
Comment (jinjahninjah) [inline structured_experiments/2022.07.19.notebook_template/experiment.MD]: Just fill in the TODOs here and your all set 👍

**Review by jinjahninjah — CHANGES_REQUESTED** [review]
Great Job! This is very helpful in giving us a direction to go in for preprocessing. Just fill in your TODOs in the readme.md and you're all set 👍

**Review by nlittlepoole — CHANGES_REQUESTED** [review]
you need to un commit all these test images because they are already tracked via LFS. otherwise everything else is good

**Review by nlittlepoole — DISMISSED** [review]
🚢

File: structured_experiments/2022.07.19.notebook_template/experiment.MD (hunk 1 above)
Comment (Samuel-Egbert31415) [reply structured_experiments/2022.07.19.notebook_template/experiment.MD]: Got it done!

**Review by jinjahninjah — APPROVED** [review]

**Review by nlittlepoole — APPROVED** [review]

**Pull request merged**